Cabal/src/Distribution/PackageDescription/Check.hs

```diff
@@ -2150,7 +2150,8 @@ fileExtensionSupportedLanguage path =
 --
 -- >>> let test fp = putStrLn $ show (isGoodRelativeDirectoryPath fp) ++ "; " ++ show (isGoodRelativeFilePath fp)
 --
--- >>> test "foo/bar/quu"
+-- >>> traverse_ test ["foo/bar/quu", "a/b.hs"]
```

Comment:
please add the `foo./bar.hs` example as well

Reply:
Sounds good!  Was just waiting to see what the verdict was on whether it should throw an error or not; sounds like "not".